website/project/metadata/osf-preregistration.json

```diff
@@ -0,0 +1,381 @@
+{
+    "name": "OSF Preregistration",
+    "version": 2,
+    "active": false,
+    "config": {
+	"hasFiles": true,
+        "fulfills": [{
+            "name": "OSF Preregistration",
+            "info": "https://cos.io/prereg/"
+        }]
+    },
+    "description": "OSF Preregistration (pre-data collection): You will be asked a series of questions to ensure that your sampling, design, and analysis plans are solidified prior to beginning your study.",
+    "pages": [{
+        "id": "page1",
+        "title": "Study Information",
+        "questions": [{
+            "qid": "q1",
+            "nav": "Title",
+            "type": "string",
+            "format": "text",
+            "title": "Title",
+            "description": "Provide the working title of your study. It is helpful if this is the same title that you submit for publication of your final manuscript, but it is not a requirement.",
```

Comment:
From the [requirements doc](https://docs.google.com/document/d/16FyCXm2dU99-RnL_riCLiLFKDlgi__gwqHmfLqRS6cc/edit) for this form, should be: "Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement."